playbooks/includes/initialize-first-node-in-cluster.yml

```diff
@@ -37,16 +37,14 @@
           curl  -u "{{couchbase_cluster.security.admin_user}}":"{{couchbase_cluster.security.admin_pwd}}" -v -X POST "{{base_uri}}"/pools/default \
           -d 'memoryQuota={{couchbase_cluster.memory_quota.data}}' \
           -d 'indexMemoryQuota={{couchbase_cluster.memory_quota.index}}' \
-          -d 'ftsMemoryQuota={{couchbase_cluster.memory_quota.search}}'
+          -d 'ftsMemoryQuota={{couchbase_cluster.memory_quota.search}}' \
+          -d 'eventingMemoryQuota={{couchbase_cluster.memory_quota.eventing}}' \
+          -d 'cbasMemoryQuota={{couchbase_cluster.memory_quota.analytics}}'
       - name: Set up Administrator Credentials
         command: >-
           curl  -u "{{couchbase_cluster.security.admin_user}}":"{{couchbase_cluster.security.admin_pwd}}" -v -X POST "{{base_uri}}"/settings/web \
           -d 'password={{couchbase_cluster.security.admin_pwd}}&username={{couchbase_cluster.security.admin_user}}&port={{couchbase_cluster.security.port}}'
 
-      - name: Play to loop over buckets and include the bucket config REST API call
```

Comment:
@davisChapmanCouchbase was there a defect with the bucket creation in this play, or was it moved out into its own playbook for another reason?

Reply:
@davisChapmanCouchbase never mind my previous question....I see your comments in the commit that explain this. Thanks for helping out!